src/main/java/foxtrot/uniform/charlie/kilo/service/implementation/dto/UserRequestMapper.java

```diff
@@ -0,0 +1,16 @@
+package foxtrot.uniform.charlie.kilo.service.implementation.dto;
+
+import foxtrot.uniform.charlie.kilo.model.User;
+import foxtrot.uniform.charlie.kilo.model.dto.UserRequestDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserRequestMapper implements MapperFromDto<UserRequestDto, User> {
```

Comment:
do we need this mapper?

Reply:
![image](https://user-images.githubusercontent.com/65593125/108200540-46c65b80-7127-11eb-9317-afa4815d508c.png)
I'm using it in registration, I think there is no matter how I exact implemented it since there will be Spring Security anyway, right?

Reply:
Now I get it.